db/migrate/20190401024406_create_guests.rb

```diff
@@ -4,9 +4,14 @@ def change
       t.string :firstname
       t.string :lastname
       t.string :email
+      t.string :occupation => 'other'
+      t.string :department
+      t.string :gender
       t.references :event, index: true
-
       t.timestamps
     end
+    SystemSetting.create  occupation:  'other',
```

Comment:
this, as far as I can tell, does nothing and is copy/pasted from the tutorial on activerecord migrations.